### ISSUE TITLE
Files not visible through MTP

### ISSUE THREAD

**Issue by martinpitt** [gsantner/markor#210]
#### General information
 - App version: 0.3.4 (installed through F-Droid)
 - System: Sony Xperia X Compact, Android 8.0.0

#### Description
When keeping the defaults, Markor will store files in /storage/emulated/0/Documents/markor/ . These are visible in Android's builtin file manager ("Dateien" App), but not through MTP, the "Documents" folder simply does not seem to exist.

When deleting the folder, creating it through MTP, and putting a few *.md files into them, Markor *does* see them, and editing them causes the files to get changed correctly. But after that, when creating a new note (or todo.txt or whatever), that particular file is again not visible through MTP. It is like Markor-created files have a special flag/owner/whatever that hides them from MTP.

I also tried to point Markor's storage path to the SD card (a path like /storage/1234-5678/Dokumente/markor/), which I would actually prefer for easier syncing. But with that, Markor seems unable to create or edit *anything* in that folder, any document creation simply goes into the void without error message.

I wonder if that is just me, or reproducible by someone else on a different model?

**Comment by gsantner** [comment]
Just google a little, thats not an markor issue. <https://encrypted.google.com/search?q=mtp files not visible>.

TL;DR: Don't use MTP.

**Comment by martinpitt** [comment]
Oops, thank you! I'll use syncthing instead.

I'll file a separate issue about SD card directories not working.

**Comment by gsantner** [comment]
not needed - if the filed are accessable by the app, markor can access zhdm. If not and it is not possibleby due some os/sw/hw restriction it cant.
